.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -13,6 +13,9 @@ name: Week 1 Step 4
 
 ### Building our first function ⚡[HackerVoice]
 
+### ✅  Tasks:
+- [ ]  Create a trigger that gets content from request parameters.
```

Comment:
This is an example where there is too much in drop downs. Try taking some content out as an "intro" to each section